staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/aes/aes.go

```diff
@@ -48,6 +52,64 @@ func NewGCMTransformer(block cipher.Block) value.Transformer {
 	return &gcm{block: block}
 }
 
+// NewGCMTransformerFromConfig takes a configuration and creates a prefix transformer to
+// handle AEAD encryption for multiple keys
+// Example configuration:
+// keys:
+//   - name: key1
+//     file: key1.txt
```

Comment:
1. Bytes in a file will be base64 encoded.
2. The configuration files are privileged and we have secrets embedded in them in other places. They have very limited accessibility because of their confidentiality.